src/App.jsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+
+    <Sum
+      a={2}
+      b={3}
+    />
+
+    <Sum
+      a={-5}
+      b={5}
+    />
+
+    <Sum
+      a={10}
+      b={0}
+    />
+
+    <Sum
+      a={0}
+      b={5}
+    />
+
+    <Sum
+      a={0}
+      b={5}
+    />
```

Comment:
```suggestion
    <Sum a={0} b={0} />
```
In last one should be no params at all
![image](https://user-images.githubusercontent.com/96409563/227144432-65882d57-d1ed-46dc-ac15-263e47f92a36.png)
![image](https://user-images.githubusercontent.com/96409563/227144527-43416a05-fa93-4bb5-b647-75fbd3563405.png)